management/connection.go

```diff
@@ -1003,6 +1004,13 @@ type ConnectionOptionsSAMLSigningKey struct {
 	Cert *string `json:"cert,omitempty"`
 }
 
+// ConnectionOptionsSAMLDecryptionKey is used to configure the
+// DecryptionKey settings on a ConnectionOptionsSAML.
+type ConnectionOptionsSAMLDecryptionKey struct {
```

Comment:
These contain exactly the same properties as the signing key, however I opted to duplicate that struct to increase clarity of the struct needed for this field. 

A potential option would be to have just 1 struct named ConnectionOptionsSAMLKey or similar and then reuse that on both signing key and decryption key fields. However that would be a breaking change. Lemme know what you think?

Reply:
I agree, best to keep them separate.